test/sol5/kyberDao.js

```diff
@@ -9,7 +9,7 @@ const Helper = require("../helper.js");
 const BN = web3.utils.BN;
 
 const { precisionUnits, zeroAddress } = require("../helper.js");
-const { expectEvent, expectRevert } = require('@openzeppelin/test-helpers');
+const { expectEvent, expectRevert, time } = require('@openzeppelin/test-helpers');
```

Comment:
nice if time only in helper
so if later we have issues, easy to change it back.
so all block increase or advance to can be done using our helper.

now its inconsistent.

Reply:
change as request